scripts/seedDB.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+const db = require("../models");
+mongoose.Promise = global.Promise;
+
+// This file empties the podcasts collection and inserts the podcasts below
+
+mongoose.connect(
```

Comment:
Dont you need to move this to the server.js file? We wont always have seed data but we will always need to spin up the db